plugins/inputs/logfile/fdlimit/fdlimit_windows.go

```diff
@@ -0,0 +1,16 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build windows
+// +build windows
+
+package fdlimit
+
+// On Windows, Go uses the CreateFile API, which is limited to 16K files; therefore, non-changeable from within a running process
+// MySQL has encountered the same issue https://bugs.mysql.com/bug.php?id=24509
```

Comment:
Reading through that link, and looking around online, that is 16384 handles for network communication as a default on Windows machines, which can be changed. Not necessarily file handlers that can be open at once. 
![Screenshot 2022-06-16 230800](https://user-images.githubusercontent.com/17056456/174217261-3ab80196-93c6-455b-82e4-c7f16eb0fc4e.png) 

I have over 80k file handles open (for some reason)

Running `net config server` on my desktop shows:
```
Maximum open files per session        16384
```

I am fine with leaving it at 16k, but the reasoning was not concrete enough.